src/SOS/SOS.Extensions/ConsoleServiceFromDebuggerServices.cs

```diff
@@ -27,12 +28,20 @@ public ConsoleServiceFromDebuggerServices(DebuggerServices debuggerServices)
 
         public void WriteDml(string text) => _debuggerServices.OutputDmlString(DEBUG_OUTPUT.NORMAL, text);
 
+        public void WriteDmlExec(string text, string cmd)
```

Comment:
Does the `dumpasync` command need to use this?  When Stephen Toub rewrote it, he added the DML support.  You don't have to do in your PR if this is necessary, but we should track it.

Reply:
Yep, I plan to update !dumpasync when I figure submit changes to fix DML output not showing up for extension commands.  I'll tackle that with the DML checkin.